bin/q

```diff
@@ -1430,19 +1435,19 @@ def quote_minimal_func(output_delimiter,v):
     if v is None:
         return v
     t = type(v)
-    if t == str or t == unicode and ((output_delimiter in v) or ('"' in v)):
+    if t == str or t == six.text_type and ((output_delimiter in v) or ('"' in v)):
```

Comment:
```suggestion
    if (t == str or t == six.text_type) and ((output_delimiter in v) or ('"' in v)):
```
This is a bug that i've found when experimenting with python 3 changes. If fixes the evaluation order to the expected one (unrelated specifically to p3)